scripts/run_integration.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+readonly SCRIPT_FOLDER=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+readonly PROJECT_FOLDER="${SCRIPT_FOLDER}/.."
+readonly ARCHIVE_FOLDER=~/archives
+readonly QEMU_INSTALL=${ARCHIVE_FOLDER}/qemu
+readonly DEFAULT_CMAKE_ARGS=" -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=ON"
+
+function extract() {
+  case $1 in
+    *.tar.bz2)   tar xjf $1    ;;
+    *.tar.xz)    tar xJf $1    ;;
+    *.tar.gz)    tar xzf $1    ;;
+    *)
+      echo "don't know how to extract '$1'..."
+      exit 1
+  esac
+}
+
+function unpackifnotexists() {
+  mkdir -p ${ARCHIVE_FOLDER}
+  cd ${ARCHIVE_FOLDER}
+  local URL=$1
+  local DESTINATION=`pwd`/$2
+  if [[  ! -d "${DESTINATION}" ]] ; then
+    local ARCHIVE_NAME=`echo ${URL} | sed 's/.*\///'`
+    test -f ${ARCHIVE_NAME} || wget ${URL}
```

Comment:
I would use -q, otherwise in travis.ci log you could have tons of progress bar lines (and reach the 4Mb limit).
cf https://api.travis-ci.org/v3/job/341333288/log.txt